src/cascadia/TerminalSettingsEditor/Appearances.h

```diff
@@ -113,6 +113,9 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         bool ShowAllFonts() const noexcept;
         void ShowAllFonts(const bool& value);
 
+        bool ShowProportionalFontWarning() const noexcept;
```

Comment:
Could we have just done a `WINRT_OBSERVABLE_PROPERTY(bool, ShowProportionalFontWarning, _PropertyChangedHandlers);` to avoid some boilerplate?

Reply:
I think so. I started with that. :|